backend/src/main/java/com/cruru/auth/service/AuthService.java

```diff
@@ -77,25 +73,25 @@ private void checkRefreshTokenExists(String refreshToken) {
         }
     }
 
-    private TokenResponse rotateTokens(Member member) {
-        Token accessToken = createAccessToken(member);
-        Token refreshToken = rotateRefreshToken(member);
+    private TokenResponse rotateTokens(String email, MemberRole role) {
+        Token accessToken = createAccessToken(email, role);
+        Token refreshToken = rotateRefreshToken(email, role);
         return new TokenResponse(accessToken.getToken(), refreshToken.getToken());
     }
 
-    private Token rotateRefreshToken(Member member) {
-        Map<String, Object> claims = getClaims(member);
+    private Token rotateRefreshToken(String email, MemberRole role) {
+        Map<String, Object> claims = getClaims(email, role);
         String token = tokenProvider.createToken(claims, tokenProperties.refreshExpireLength());
-        RefreshToken refreshToken = new RefreshToken(token, member);
+        RefreshToken refreshToken = new RefreshToken(token, email);
 
-        refreshTokenRepository.updateRefreshTokenByMember(refreshToken.getToken(), member);
+        tokenRedisClient.saveToken(email, refreshToken.getToken());
         return refreshToken;
     }
 
-    private Map<String, Object> getClaims(Member member) {
+    private Map<String, Object> getClaims(String email, MemberRole role) {
         Map<String, Object> claims = new HashMap<>();
-        claims.put(EMAIL_CLAIM, member.getEmail());
-        claims.put(ROLE_CLAIM, member.getRole().name());
+        claims.put(EMAIL_CLAIM, email);
+        claims.put(ROLE_CLAIM, role.name());
```

Comment:
```suggestion
    return Map.of(
        EMAIL_CLAIM, email,
        ROLE_CLAIM, role.name()
    );
```
claim 변경될 정보 아니니까 불변 어떤가용?
필수 반영사항 아닙니다~